client/container_exec.go

```diff
@@ -43,8 +43,19 @@ func (cli *Client) ContainerExecCreate(ctx context.Context, containerID string,
 	return response, err
 }
 
+// ExecStartOptions is a temp struct used by execStart
+// Config fields is part of ExecConfig in runconfig package
+type ExecStartOptions struct {
+	// ExecStart will first check if it's detached
+	Detach bool
+	// Check if there's a tty
+	Tty bool
```

Comment:
I need to look at the options; both `ContainerExecStart` and `ContainerExecAttach` use the `/start/` endpoint, but only `ContainerExecAttach` handles attaching, so likely this option shouldn't be available in both.